### PR TITLE
fix: show images low quality

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [https://www.notion.so/artsy/Images-for-shows-in-Shows-For-You-rail-appear-low-resolution-11acab0764a080ff98b3c0f80040f460?pvs=4] 

### Description

This PR fixes an issue with the shows rail where the images are very low quality

| Before | After |
|--------|--------|
| Simulator Screenshot - iPhone 16 Pro - 2024-10-11 at 11 10 46 | Simulator Screenshot - iPhone 16 Pro - 2024-10-11 at 11 11 50 |

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- fix shows rail images poor quality on home - mounir

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (fix shows rail images poor quality on home - mounir)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against f74df114d5a7c1a76d195d9be6c5218dddf1daed
</p>

File: src/app/Components/ShowCard.tsx [hunk 1]
```diff
@@ -97,7 +97,7 @@ export const ShowCardContainer = createFragmentContainer(ShowCard, {
       formattedEndAt: endAt(format: "MMM D")
       href
       metaImage {
-        url(version: "small")
+        url(version: "large")
```
Comment (gkartalis) [inline src/app/Components/ShowCard.tsx:100]: 👏👏

**Review by gkartalis — APPROVED** [review]

File: src/app/Components/ShowCard.tsx (hunk 1 above)
Comment (dblandin) [reply src/app/Components/ShowCard.tsx:100]: Wow, why was this ever small? 🤦🏼 

**suggestion**:  Can we resize from the the larger version instead?

```graphql
metaImage {
  cropped(height: $height, width: $width version: "larger") {
    url
  }
}
```

Comment (dblandin) [reply src/app/Components/ShowCard.tsx:100]: Ahhh, I see some of these Image components do resizing internally. So maybe no need to explicitly do that here in the query. But still think we should bump from `large` to `larger`.

Comment (dblandin) [reply src/app/Components/ShowCard.tsx:100]: I went ahead and PR'd this [here](https://github.com/artsy/eigen/pull/10947).

**Pull request merged**